ucm2/USB-Audio/Realtek/ALC4080-HiFi.conf

```diff
@@ -0,0 +1,57 @@
+SectionDevice."Speaker" {
+	Comment "Speakers"
+	Value {
+		PlaybackChannels 8
+		PlaybackPriority 200
+		PlaybackPCM "hw:${CardId}"
+		JackControl "Speaker - Output Jack"
+		PlaybackMixerElem "Speaker"
+	}
+}
+
+SectionDevice."Headphones" {
+	Comment "Front Headphones"
+	Value {
+		PlaybackPriority 300
+		PlaybackPCM "hw:${CardId},1"
+		JackControl "Headphone - Output Jack"
+		PlaybackMixerElem "Front Headphone"
+	}
+}
+
+SectionDevice."SPDIF" {
+	Comment "S/PDIF Out"
+	Value {
+		PlaybackPriority 100
+		PlaybackPCM "hw:${CardId},3"
+		PlaybackMixerElem "IEC958"
```

Comment:
This should be `hw:${CardId},2`.

Reply:
On MSI MAG X570S Torpedo Max: `hw:${CardId},3` outputs to S/PDIF (tested)

Reply:
I'm not sure what to do here. Maybe the ASUS board has a different layout? Which board did you try this on @Tatsh?

Reply:
It's definitely 2 on ASUS ROG Maximus XIII. What you could do is have this be conditional based on `RegexMatch` (not tested below).

```
Define SPDIFIndex "3"

If.asus-rog-maximus-xii {
    Condition {
      Type RegexMatch
       String "${CardComponents}"
      Regex "USB(0b05:1996)"
    }
    True.SPDIFIndex "2"
}
```

Then refer to `${SPDIFIndex}` in `PlaybackPCM`.

Reply:
Fixed syntax:

```
Define.SPDIFIndex "3"

If.asus-rog-usb {
        Condition {
                Type RegexMatch
                String "${CardComponents}"
                Regex "USB(0b05:1996)"
        }
        True.Define.SPDIFIndex "2"
}

...

SectionDevice."SPDIF" {
        Comment "S/PDIF Out"
        Value {
                PlaybackPriority 100
                PlaybackPCM "hw:${CardId},${var:SPDIFIndex}"
                PlaybackMixerElem "IEC958"
        }
}
```

Reply:
@lordhoto SPDIFIndex is 2 on the MSI MPG Z690 EDGE(0db0:1feb) as well.

Do I need to open a new issue for this change?

Reply:
@tekstryder sorry for the late reply: yes, you'll need to open a new issue/pull request for this change. There is pull request has been closed already.